package-lock.json

```diff
@@ -34035,7 +34052,7 @@
             "requires": {
                 "extend": "^3.0.0",
                 "glob": "^7.1.1",
-                "glob-parent": "5.1.2",
+                "glob-parent": "^3.1.0",
```

Comment:
Not sure this is a valid change.
Please could you double check your version or node and npm
This looks like a downgrade

Reply:
You need
1. [Node.js](https://nodejs.org/) 16.14.2
2. [npm](https://www.npmjs.com/) 8.15.1

Reply:
Seems like these are out of sync, I have that node version and npm 8.5.0 which is what it appears to have been published with, https://nodejs.org/en/download/releases/ is that right?

Reply:
you need the latest version of npm, as npm 8.5.0 has a bug when using the overrides feature.

Reply:
& yes, they are out of sync, either we go with thie route or use yarn, i mentioend this in standup & there was no preference to use yarn, hence i didn't want to make a bigger change by changing the lock file from npm to yarn

This was to fix the CG issues.

Reply:
Oh, I understand, ok thanks